packages/templates/src/raw/kitchen-stories/template.json

```
@@ -0,0 +1,6358 @@
+          },
+          "fontSize": 26,
+          "backgroundColor": {
+            "color": {
+              "r": 196,
+              "g": 196,
+              "b": 196
+            }
+          },
+          "lineHeight": 1.14,
+          "textAlign": "center",
+          "padding": {
+            "vertical": 0,
+            "horizontal": 0,
+            "locked": true
+          },
+          "content": "<span style=\"font-weight: 700; color: #fff\">Add the beans</span>",
+          "fontWeight": 400,
+          "borderRadius": {
+            "locked": true,
+            "topLeft": 0,
+            "topRight": 0,
+            "bottomRight": 0,
+            "bottomLeft": 0
+          },
+          "width": 229,
+          "height": 35,
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "basedOn": "d9e02efb-411c-4e80-8c30-4eb9afeeb3ce",
+          "id": "e2d64143-9607-4e62-8dce-c394ec3d38e0",
+          "x": 92,
+          "y": 19,
+          "type": "text"
+        }
+      ],
+      "backgroundOverlay": "none",
+      "animations": [],
+      "backgroundColor": {
+        "color": {
+          "r": 255,
+          "g": 255,
+          "b": 255
+        }
+      },
+      "type": "page",
+      "id": "7a1236ce-eeb5-468b-94ff-63491ddea1f8",
+      "defaultBackgroundElement": {
+        "opacity": 100,
+        "flip": {
+          "vertical": false,
+          "horizontal": false
+        },
+        "rotationAngle": 0,
+        "lockAspectRatio": true,
+        "backgroundColor": {
+          "color": {
+            "r": 255,
+            "g": 255,
+            "b": 255
+          }
+        },
+        "x": 1,
+        "y": 1,
+        "width": 1,
+        "height": 1,
+        "mask": {
+          "type": "rectangle"
+        },
+        "isBackground": true,
+        "isDefaultBackground": true,
+        "id": "ab1771cd-eb71-43b1-ab77-4e5dddba90d7",
+        "type": "shape"
+      },
+      "pageTemplateType": ""
+    },
+    {
+      "elements": [
+        {
+          "opacity": 100,
+          "flip": {
+            "vertical": false,
+            "horizontal": false
+          },
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "backgroundColor": {
+            "color": {
+              "r": 255,
+              "g": 255,
+              "b": 255
+            }
+          },
+          "x": -98,
+          "y": -51,
+          "width": 608,
+          "height": 720,
+          "mask": {
+            "type": "rectangle"
+          },
+          "isBackground": true,
+          "id": "52bac944-215a-4df7-a193-d358702a4bdd",
+          "type": "video",
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "resource": {
+            "type": "video",
+            "mimeType": "video/mp4",
+            "creationDate": "2021-05-10T12:53:21",
+            "src": "__WEB_STORIES_TEMPLATE_BASE_URL__/images/templates/kitchen-stories/page11_bg_videon.mp4",
```

```suggestion
            "src": "__WEB_STORIES_TEMPLATE_BASE_URL__/images/templates/kitchen-stories/page11_bg_video.mp4",
```

@divyarajmasani FYI, just one small fix

---

I've pushed the fix. Thank you for looking into it,